soc/silabs/silabs_s2/soc.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2025 Silicon Laboratories Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Register access macros for Silabs Series 2 SoCs
+ *
+ */
+
+#ifndef SILABS_S2_SOC_H_
+#define SILABS_S2_SOC_H_
+
+#ifndef _ASMLANGUAGE
+
+#include <em_common.h>
+
+#endif /* !_ASMLANGUAGE */
+
+#endif /* SILABS_S2_SOC_H_ */
```

Comment:
Because of the "Don't Repeat Yourself" rule, I suggest to not append comments after `#endif` when the condition is trivial.

Reply:
Done.